data/templates/dnsmasq/plain/dnsmasq.conf

```diff
@@ -1,6 +1,9 @@
 domain-needed
 expand-hosts
 
+log-facility=/var/log/dnsmasq.log
+log-queries
```

Comment:
To have a log of dnsmasq, you don't need that option.
And this option is going to log every dns request, with the name of the domain requested and the IP of who ask to it.
That means, it's quite tricky regarding the privacy of the users who use the dns.

Also, this option without a logrotate would be a suicide ! I stucked my own server with the 11G log in a few days.

Please have a look here http://www.thekelleys.org.uk/dnsmasq/docs/dnsmasq-man.html

Reply:
Same point of view than Maniack